### PR TITLE
add first Thursday festival 

### PR DESCRIPTION
**Pull request by Medina88**

### CODE REVIEW

File: content/venues/2022/07-27/first-thursdays-festival/readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+dateStart: 2022-09-01T21:00:17.165Z
```
Comment (bherr2) [inline content/venues/2022/07-27/first-thursdays-festival/readme.md:2]: Date and folder names do not align. Please correct whichever is the right one.

Comment (bherr2) [reply content/venues/2022/07-27/first-thursdays-festival/readme.md:2]: @Medina88 did you see this?

File: content/venues/2022/09-29/technology-petting-zoo/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+dateStart: 2022-09-27T15:00:32.120Z
```
Comment (bherr2) [inline content/venues/2022/09-29/technology-petting-zoo/readme.md:2]: @Medina88  date says 9/27, folder says 9/29?

File: content/venues/2022/07-27/first-thursdays-festival/readme.md (hunk 1 above)
Comment (Medina88) [reply content/venues/2022/07-27/first-thursdays-festival/readme.md]: corrected the folders, all should be good now.

**Review by bherr2 — APPROVED** [review]

**Comment by bherr2** [comment]
@Medina88 merged!

**Pull request merged**